financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthViewModel.kt

```diff
@@ -225,6 +227,45 @@ internal class PartnerAuthViewModel @Inject constructor(
         navigationManager.navigate(NavigationDirections.manualEntry)
     }
 
+    fun onClickableTextClick(uri: String) {
+        // if clicked uri contains an eventName query param, track click event.
+        viewModelScope.launch {
+            uriUtils.getQueryParameter(uri, "eventName")?.let { eventName ->
+                eventTracker.track(
+                    FinancialConnectionsEvent.Click(
+                        eventName,
+                        pane = Pane.PARTNER_AUTH
+                    )
+                )
+            }
+        }
+        val date = Date()
```

Comment:
nit: Inline this so `null` on 264 case doesn't need to create one

Reply:
https://github.com/stripe/stripe-android/pull/5962/commits/83ce11d97d94dd8a8fb2d72c34b2bdf0909ccc40